domain/blockdag/dag.go

```diff
@@ -281,6 +281,15 @@ func (dag *BlockDAG) SelectedTipBlueScore() uint64 {
 	return dag.selectedTip().blueScore
 }
 
+// VirtualBlues returns the blues of the current virtual block
+func (dag *BlockDAG) VirtualBlues() []*daghash.Hash {
```

Comment:
Please rename this to VirtualBlueHashes.

Reply:
Done